tests/unit_and_functional/test_workflow_execution.py

```diff
@@ -18,7 +18,7 @@
 Test Bodywork workflow execution.
 """
 from pathlib import Path
-from unittest.mock import MagicMock, patch, ANY
+from unittest.mock import MagicMock, patch, ANY, PropertyMock
```

Comment:
`PropertyMock` isn't used anywhere.

I was about to ask you to teach me about it...

Reply:
Good spot! I did use it and then didn't need it anymore.